wire-ios/Wire-iOS/Sources/Authentication/Coordinator/AuthenticationCoordinator.swift

```diff
@@ -375,8 +367,6 @@ extension AuthenticationCoordinator: AuthenticationActioner, SessionManagerCreat
                 activateNetworkSessions { [weak self] _ in
                     self?.startCompanyLoginFlowIfPossible(linkCode: code)
                 }
-            case .startSSOFlow:
```

Comment:
It's possible that this might be used shortly and should be reverted

Reply:
yes 👍

Reply:
I reverted that commit